int8[] actions
---
observation[] obs
int32[] rewards
bool[] episodes_done
